serenata_toolbox/chamber_of_deputies/dataset.py

```diff
@@ -137,3 +132,6 @@ def __translate_file(self, csv_path):
                     encoding='utf-8')
 
         return output_file_path
+
+    def __parse_float(self, string):
```

Comment:
Just because preceding a method with a double underscore will make it inaccessible from outside of the object.

```
In [1]: class Object:
   ...:     def foo(self):
   ...:         return 1
   ...:     def _foo(self):
   ...:         return 2
   ...:     def __foo(self):
   ...:         return 3
   ...:

In [2]: Object().foo()
Out[2]: 1

In [3]: Object()._foo()
Out[3]: 2

In [4]: Object().__foo()
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-7-b254b9c59e17> in <module>()
----> 1 Object().__foo()

AttributeError: 'Object' object has no attribute '__foo'
```